src/fluid_ele/4C_fluid_ele.cpp

```diff
@@ -63,7 +63,23 @@ void Discret::Elements::FluidType::nodal_block_information(
 Core::LinAlg::SerialDenseMatrix Discret::Elements::FluidType::compute_null_space(
     Core::Nodes::Node& node, const double* x0, const int numdof, const int dimnsp)
 {
-  return FLD::compute_fluid_null_space(numdof, dimnsp);
+  switch (numdof)
+  {
+    case 3:
+      // 2D fluid
+      return FLD::compute_fluid_null_space<3>();
```

Comment:
Why not simply return an identity matrix here? If you believe it helps to have this function that is OK with me. Although it seems the helper is not really doing anything other than constructing an identity matrix.